docs/storage.rst

```diff
@@ -97,6 +97,31 @@ Here is an example using a `RedisCache`: ::
 This is primarily a proof of concept, so please file bugs if there is
 a better method for utilizing redis as a cache.
 
+UWSGICache
```

Comment:
My hope is that you can host this backend in its own repo, in which case, you can move the summary to the `Third-Party Cache Providers` and provide a link to help folks find it!

Reply:
Will do !